mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/events/ElasticSearchListeningMessageSearchIndex.java

```diff
@@ -117,18 +117,19 @@ public void add(MailboxSession session, Mailbox mailbox, MailboxMessage message)
             indexer.indexMessage(indexIdFor(mailbox, message.getUid()), messageToElasticSearchJson.convertToJson(message, ImmutableList.of(session.getUser())));
         } catch (Exception e) {
             try {
-                LOGGER.warn("Indexing mailbox {}-{} of user {} on message {} without attachments ",
+                LOGGER.warn(String.format("Indexing mailbox %s-%s of user %s on message %s without attachments ",
```

Comment:
It depends of the number of paramenters IE:

```
LOGGER.warn("Message {} with one more {} then I will forgot my second parameter", firstParam, e);
```

Will produce:

```
[main] WARN logger - Message value1 with one more java.lang.Exception then I will forgot my second parameter
```

Which is very dangerous! It behaves well with only one placeholder however.

I also fear the behaviour depends of the SLF4J implemetation.

Mixing Exception and params is : 
 - **dangerous**
 - **unclear**
 - **easily avoided**

That's 3 good reasons to run away from it, and not playing with fire!!!